packages/react-components/react-menu/stories/src/Menu/MenuMultilineItems.stories.tsx

```diff
@@ -0,0 +1,347 @@
+          <MenuItem>New folder</MenuItem>
+        </MenuList>
+      </MenuPopover>
+    </Menu>
+  </MenuGroup>
+);
+
+const SelectableTextOnly = () => (
+  <MenuGroup>
+    <MenuGroupHeader>Selectable text only</MenuGroupHeader>
+    <MenuItemCheckbox value="foo" name="foo">
+      Select this thing
+    </MenuItemCheckbox>
+    <MenuItemCheckbox value="foo" name="foo" subText="Selection">
+      Select this thing
+    </MenuItemCheckbox>
+  </MenuGroup>
+);
+
+const SelectableWithIcon = () => (
+  <MenuGroup>
+    <MenuGroupHeader>Selectable with Icon</MenuGroupHeader>
+    <MenuItemCheckbox value="foo" name="bar" icon={<EditIcon />}>
+      Select this thing
+    </MenuItemCheckbox>
+    <MenuItemCheckbox value="foo" name="bar" subText="Selection" icon={<EditIcon />}>
+      Select this thing
+    </MenuItemCheckbox>
+  </MenuGroup>
+);
+
+const SelectableWithIconAndSecondary = () => (
+  <MenuGroup>
+    <MenuGroupHeader>Selectable with icon and secondary</MenuGroupHeader>
+    <MenuItemCheckbox icon={<EditIcon />} value="foo" name="bac" secondaryContent="Ctrl+Spacebar">
+      Select this thing
+    </MenuItemCheckbox>
+    <MenuItemCheckbox icon={<EditIcon />} value="foo" name="bac" subText="Selection" secondaryContent="Ctrl+Spacebar">
+      Select this thing
+    </MenuItemCheckbox>
+  </MenuGroup>
+);
+
+const SelectableWithSecondary = () => (
+  <MenuGroup>
+    <MenuGroupHeader>Selectable with secondary</MenuGroupHeader>
+    <MenuItemCheckbox value="foo" name="baz" secondaryContent="Ctrl+Spacebar">
+      Select this thing
+    </MenuItemCheckbox>
+    <MenuItemCheckbox value="foo" name="baz" subText="Selection" secondaryContent="Ctrl+Spacebar">
+      Select this thing
+    </MenuItemCheckbox>
+  </MenuGroup>
+);
+
+const SwitchTextOnly = () => (
+  <MenuGroup>
+    <MenuGroupHeader>SwitchTextOnly</MenuGroupHeader>
+    <MenuItemSwitch value="foo" name="switch">
+      Select this thing
+    </MenuItemSwitch>
+    <MenuItemSwitch value="foo" name="switch" subText="Selection">
+      Select this thing
+    </MenuItemSwitch>
+  </MenuGroup>
+);
+
+const SwitchWithIcon = () => (
+  <MenuGroup>
+    <MenuGroupHeader>SwitchTextOnly</MenuGroupHeader>
+    <MenuItemSwitch icon={<EditIcon />} value="foo" name="switch1">
+      Select this thing
+    </MenuItemSwitch>
+    <MenuItemSwitch icon={<EditIcon />} value="foo" name="switch1" subText="Selection">
+      Select this thing
+    </MenuItemSwitch>
+  </MenuGroup>
+);
+
+const SwitchWithIconAndSecondary = () => (
+  <MenuGroup>
+    <MenuGroupHeader>SwitchTextOnly</MenuGroupHeader>
+    <MenuItemSwitch icon={<EditIcon />} value="foo" name="switch2" secondaryContent="Ctrl+Spacebar">
+      Select this thing
+    </MenuItemSwitch>
+    <MenuItemSwitch icon={<EditIcon />} value="foo" name="switch2" subText="Selection" secondaryContent="Ctrl+Spacebar">
+      Select this thing
+    </MenuItemSwitch>
+  </MenuGroup>
+);
+
+export const MultilineItems = () => {
```

Comment:
TODO - this is for playground purposes for designers, final doc example will be more compact